### ISSUE TITLE
-2. Apply to the plankton data set.

### ISSUE THREAD

**Issue by juleshenry** [juleshenry/quantum-mnist#3]
We will look at the cartesian product comparison of binary classification via the "fair" 4x4 neural net. Do the same for the quantum scenario.
We need to convert the plankton dataset to grayscale. We do so using the default configuration in Pillow. Next, we downsize colors to normalize. In the current iteration, the hyperparameters of batch_size and image size must be explored. The size of 4x4 is insufficient for the plankton dataset, so a larger quantum simulation will be needed than described in the 4x4 method. Even in 16x16, the naive FFN is not powerful. (We are here)
